### PR TITLE
added entitlements to createSessionFromID

### PR DESCRIPTION
**Pull request by jareddlc**
This adds the entitlement object to the session object that is returned from createSessionFromID

### CODE REVIEW

File: lib/auth.js [hunk 1]
```diff
@@ -263,6 +263,21 @@ Auth.createSessionFromID = function createSessionFromID(id, callback) {
 				package: body.org.package,
 				packageId: body.org.packageId
 			};
+
+			var entitlements = body.org.entitlements;
+			entitlements.id = body.org.packageId;
+			entitlements.name = body.org.package;
```
Comment (pkclark) [inline lib/auth.js]: @jareddlc nothing should be predicated on package ID or name.  Why are these needed?

File: lib/auth.js [hunk 2]
```diff
@@ -263,6 +263,21 @@ Auth.createSessionFromID = function createSessionFromID(id, callback) {
...
+			entitlements.partners = ['acs', 'analytics'];
+
+			if (body.org.limit_performance_users && body.org.limit_performance_users > 0) {
+				entitlements.partners.push('crittercism');
```
Comment (pkclark) [inline lib/auth.js]: @feons `entitlements.partners.push('crash')` as well for ACA?

Comment (feons) [inline lib/auth.js]: yes yes yes!

Comment (pkclark) [inline lib/auth.js]: @jareddlc need to check that these aren't in the partners array prior to pushing?

Comment (pkclark) [inline lib/auth.js]: @jareddlc not sure if we need to support unlimited seats for those roles, but may be worth just checking `!== 0`

Comment (feons) [inline lib/auth.js]: @pkclark @jareddlc , will `crittercism` and `crash` coexist?

Comment (pkclark) [inline lib/auth.js]: @feons @jareddlc it should, yes.

**Comment by pkclark** [comment]
LGTM.  Thanks @jareddlc
